classes/form/FormBuilderVocabulary.inc.php

```diff
@@ -54,6 +54,8 @@
  *   required: Adds an asterisk and a .required class to the element's label
  */
 
+define('INSERT_TAG_VARIABLE_TYPE_PLAIN_TEXT', 'PLAIN_TEXT');
```

Comment:
If you use `define_exposed`, you'll be able to reference the constant from JS using `$.pkp.cons.INSERT_TAG_VARIABLE_TYPE_PLAIN_TEXT` (OTOH) rather than having to redefine it JS-side.